Tools/TestWebKitAPI/WTFTestUtilities.h

```diff
@@ -30,33 +30,20 @@
 
 #pragma once
 
-#ifdef WTF_STRINGTYPEADAPTER_COPIED_WTF_STRING
-#error "WTFStringUtilities.h must be included before <wtf/StringConcatenate.h>"
-#else
-#define WTF_STRINGTYPEADAPTER_COPIED_WTF_STRING() (++wtfStringCopyCount)
-extern int wtfStringCopyCount;
-#endif
-
-#include <wtf/Assertions.h>
 #include <wtf/text/CString.h>
 #include <wtf/text/StringBuilder.h>
 #include <wtf/text/WTFString.h>
 
-namespace WTF {
-
-inline std::ostream& operator<<(std::ostream& os, const String& string)
-{
-    return os << string.utf8().data();
-}
-
-}
+namespace TestWebKitAPI {
 
 template<size_t length>
-static String utf16String(const char16_t (&url)[length])
```

Comment:
A bit weird to see a remove `static` declaration in a header file, aren't we going to have multiple definitions of this function now?

Reply:
It's a template, those are defined like that.
static template definition means each compilation unit that uses the function gets a compilation unit local template instantiation.

Reply:
🤦‍♂️ Missed the `template<size_t length>` on the the line above.